Jenkinsfile_k8s

```diff
@@ -55,6 +55,11 @@ spec:
     - name: "jnlp"
       image: "jenkinsciinfra/builder"
       imagePullPolicy: Always
+      resources:
+        limits: {}
```

Comment:
Is there a reason to not put limits? (I see a lot for setting limits in term of hosting tbh)